src/widgets/panel/panel.js

```diff
@@ -106,7 +129,11 @@ export default function panel({
       );
     }
 
-    const defaultTemplates = { header: '', footer: '' };
+    const defaultTemplates = {
+      header: '',
+      footer: '',
+      collapseButtonText: isPanelCollased => (isPanelCollased ? '➕' : '➖'),
```

Comment:
It keeps not working the value is an `object` not a `boolean`. You've to extract the value from the argument.